DigitalLearningSolutions.Data.Tests/Services/LoginServiceTests.cs

```diff
@@ -211,72 +198,67 @@ public void AttemptLogin_returns_inactive_centre_if_all_centres_are_inactive()
             using (new AssertionScope())
             {
                 result.LoginAttemptResult.Should().Be(LoginAttemptResult.InactiveCentre);
-                result.LogInAdmin.Should().Be(null);
-                result.LogInDelegates.Should().BeEmpty();
+                result.Accounts.AdminAccount.Should().Be(null);
+                result.Accounts.DelegateAccounts.Should().BeEmpty();
             }
         }
 
         [Test]
         public void
-            AttemptLogin_does_not_find_linked_admins_if_verified_admin_already_found_and_no_linked_delegates_and_returns_single_centre_login_result()
+            AttemptLogin_finds_linked_delegate_and_returns_single_centre_login_result()

```

Comment:
It feels more natural to call this `AttemptLogin_returns_single_centre_login_result_when_single_delegate_found` - is there a reason not to call it that? Currently it sounds like we're testing that it finds a linked delegate, but also have a line in the test `GivenLinkedDelegateAccountsFound(<single delegate>)`.

Reply:
I don't think the suggested names for these tests are better, we are testing the linked delegates cases. We start with a single admin then find linked delegate(s) and return the appropriate result now we have an admin + delegate(s).

i.e. we're testing exactly what you think it sounds like.